### PR TITLE
flowc: proxy flowc1 script is removed, flows is called directly

### PR DESCRIPTION
**Pull request by dmitry-vlasov**

### CODE REVIEW

**Comment by Eugeny48** [comment]
This will not work on Windows, I use Windows.

**Comment by Eugeny48** [comment]
It works for me, but just too much output:
```
Current directory: x:\innovation\components\master_key
Running x:\flow9\bin\flowc server=0 file=master_key\share_key\lib\share_key_sql.flow
    JAVA_HOME has been set to:

        C:\Program Files\Java\jdk1.8.0_211

Flow compiler (3rd generation)



Processing 'tools/flowc/flowc' on server.
Processing 'tools/flowc/flowc' on server.
7.33s
7.34s



Flow compiler (3rd generation)

0.48s
```

File: bin/flowc1.bat [hunk 1]
```diff
@@ -1,11 +1,46 @@
...
+pushd .
+
+for /d %%i in ("%ProgramFiles%\Java\jdk1.8*") do (set Located=%%i)
+rem check if JDK was located
+if "%Located%"=="" goto else
+rem if JDK located display message to user
+rem update %JAVA_HOME%
+set JAVA_HOME=%Located%
+echo     JAVA_HOME has been set to:
+echo         %JAVA_HOME%
```
Comment (alstrup) [inline bin/flowc1.bat]: Omit this

File: bin/flowc1.bat [hunk 2]
```diff
@@ -1,11 +1,46 @@
...
+goto endif
+
+:else
+rem if JDK was not located
+rem if %JAVA_HOME% has been defined then use the existing value
+if "%JAVA_HOME%"=="" goto NoExistingJavaHome
+echo     Existing value of JAVA_HOME will be used:
+echo         %JAVA_HOME%
```
Comment (alstrup) [inline bin/flowc1.bat]: Omit this

File: bin/flowc1.bat [hunk 3]
```diff
@@ -1,11 +1,46 @@
...
+rem display message to the user that %JAVA_HOME% is not available
+echo     No Existing value of JAVA_HOME is available.
+echo     set JAVA_HOME=path to your jdk root
+echo     and retry
+goto endif
+
+:endif
+
+set JAVAC=%JAVA_HOME%\bin\javac
+set JAR=%JAVA_HOME%\bin\jar
```
Comment (alstrup) [inline bin/flowc1.bat]: Unused

**Comment by Eugeny48** [comment]
Now it works fine

**Pull request merged**